crates/router/src/scheduler/consumer.rs

```diff
@@ -234,9 +234,9 @@ pub async fn some_error_handler<E: fmt::Display>(
     process: storage::ProcessTracker,
     error: E,
 ) -> CustomResult<(), errors::ProcessTrackerError> {
-    error!(%process.id, "Failed while executing workflow");
-    error!(%error);
-    error!(
+    logger::error!(%process.id, "Failed while executing workflow");
+    logger::error!(%error);
+    logger::error!(
```

Comment:
Could you update this block to be logging a single log event, and use one of the error messages? And maybe also rename the function to remove the `some` I added when I wrote it 😬 ?
```rust
logger::error!(pt.name = ?process.name, pt.id = %process.id, %error, "Failed while executing workflow");
```

Reply:
Sure, your suggestion looks more cleaner in this case